### DISCUSSION TITLE
On Security Onion 2.3 how do I change the docker ip subnet range?

### DISCUSSION THREAD

**Discussion by mgbach6410** [Security-Onion-Solutions/securityonion#1588] [Unsupported Versions] [answered]
I know I need to do this in the /etc/docker/daemon.json file.  At least I did in the old version.  But that ISO came with docker preinstalled allowing me to make that change before running setup.  My concern here is more related to the firewall rules that are built.  If I just change the ips will the firewall rules change too?  Or do I need to run the setup change the file and then run the setup again?

**Comment by TOoSmOotH** [comment, marked as answer]
We will put together instructions on how to do this.
